src/pybind/mgr/dashboard/tox.ini

```diff
@@ -85,7 +85,8 @@ wrap_length = 80
 # Allow similarity/code duplication detection
 jobs = 1
 dirs = . api controllers plugins services tests
-addopts = -rn --rcfile=.pylintrc --jobs={[pylint]jobs}
+disable = duplicate-code
```

Comment:
Nooo, plz. This helped a lot in the past to detect duplicate code... I guess this is needed because of the `setup_class()` method enabling request logging (duplicate code failure is triggered with 4 or more repeated lines), right?

Why not moving that functionality to the base `ControllerTestCase.setUpClass(cls, request_logging=False)` to only enable the logging on those tests that explicitly required it? It could be called as:
```py
class RequestLoggingToolTest(ControllerTestCase):
  @classmethod
  def setUpClass(cls):
    super().setUpClass(request_logging=True)

```

Reply:
Hmmm I faced a similar issue here too: https://github.com/ceph/ceph/pull/37568/commits/c9b91e01c9de925616b0f6862ca31d36370f18e7

Reply:
i refactored the code per your suggesting. but then the lint test complains that the method signature of the derived class' `setUpClass()` was different from that of parent. so i extract the method out into a `staticmethod`. again, `duplicate-code` warning! anyway, i addressed it using
```py
super().tearDownClass()
cls.disable_request_logging()
```
i knew it's far from ideal. and i don't like it. but neither do i want to explore more options for fixing it.

Reply:
Oh, weird. @tchaikov, the alternatives I see here (ranked from more to less optimal, IMHO):
1. Inline disabling the 'method not matching parent's signature' or whatever is called (since as @s0nea mentioned the disable doesn't work with duplicate-code)...
1. Increasing similarity setting `min-similarity-lines` to 5 instead of 4 in `.pylintrc`...
1. Disabling similarity check. In that case, please let's increase the job number (it was set to 1 because the similarity check required a sequential execution) 😞